src/main/java/com/google/alpollo/AnalysisHelper.java

```diff
@@ -0,0 +1,35 @@
+package com.google.alpollo;
+
+import java.io.IOException;
+import com.google.api.gax.rpc.FixedHeaderProvider;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.LanguageServiceSettings;
+import com.google.cloud.language.v1.Sentiment;
+
+/**
+ * Helper class with methods that use the Natural Language API or use the data that comes
+ * from the API.
+ */
+public class AnalysisHelper {
+  private static LanguageServiceSettings SETTINGS;
```

Comment:
This is not a constant as it is not final, use lower case for the name

Reply:
You're right, I confused static with constant.